features/discover/d2l-discover-rules.js

```diff
@@ -203,7 +204,7 @@ class EntitlementRules extends LocalizeDynamicMixin(SkeletonMixin(HypermediaStat
 	}
 
 	async _summonEntitlement() {
-		const sirenReponse = await this._getEntitlement.summon(null, true);
+		const sirenReponse = await this._getEntitlement.summon({profileCount : profileCount});
 		if (sirenReponse) {
 			const newRules = sirenReponse?.entities.filter(e => e.rel.includes(rels.rule));
```

Comment:
```suggestion
			const newRules = sirenReponse.entities.filter(e => e.rel.includes(rels.rule));
```
You already have a check for existence wrapping it